rake/spec/default/60-st2_all-services-ok_spec.rb

```diff
@@ -7,20 +7,17 @@
 
   describe 'rabbitmq' do
     subject { host(spec[:rabbitmqhost]) }
-    it { is_expected.to be_reachable }
```

Comment:
I think these tests are legit.

I intentionally leaved them here, because we can have 2 issues:
1.) When host doesn't exists at all
2.) When host reachable, but mongo or any other service is not reachable by port

They're completely different and show different problems.
Leaving both tests will help to identify issues faster, for example when we change Docker version and will have changed IPs (as it was when I switched to Docker 1.9, if you remember).